blogposts/2021/release-3.32.md

```diff
@@ -0,0 +1,53 @@
+---
+title: 'Sourcegraph 3.32 release'
+publishDate: 2021-09-23T10:00-07:00
+description: 'Sourcegraph 3.32 introduces Code Insights in Public Beta, a new search sidebar for navigating revisions, improved search results caching, and custom LSIF data retention policies.'
+tags: [blog, release]
+slug: 'release/3.32'
+published: false
+heroImage:
```

Comment:
@iskyOS - here is the hero template  https://www.figma.com/file/BRedkGL1VSHDOjM2ci4L0Q/SG_Evergreen_Blog_Template?node-id=0%3A1